web/modules/custom/asset_distribution/asset_distribution.module

```diff
@@ -156,6 +157,31 @@ function asset_distribution_rdf_entity_presave(RdfInterface $distribution) {
 }
 
 /**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function asset_distribution_rdf_entity_insert(EntityInterface $entity) {
```

Comment:
Maybe the cross invocation of hook_update could have been avoided by using a hook_presave().